product_brand/__manifest__.py

```diff
@@ -13,7 +13,7 @@
     'summary': "Product Brand Manager",
     'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
               ', MONK Software, Odoo Community Association (OCA)'
-              ', SerpentCS Pvt. Ltd.',
+              ', SerpentCS Pvt. Ltd., Amaris',
```

Comment:
Fixing the pylint doesn't make you author of the module, please remove this and add yourself to the contributors sections if you wish, however you haven't change/improved any functionality of the module, IMHO you shouldn't either. 

Same applies for all the copyright you added on the rest of files.